keycloak/themes/uid2-theme/email/messages/messages_en.properties

```diff
@@ -4,5 +4,5 @@ emailVerificationBody=Please verify your email address\n\nClick the link below t
 passwordResetSubject=Reset password
 passwordResetBodyHtml=<p> Hi {0}, </p> <p> We received a request to reset your password for your account. </p> <p> Please use the button below to reset your password. </p> <div class="footer"> <a href="{1}" class="button" >Reset Password</a > </div>
 passwordResetBody=Hi {0}, We received a request to reset your password for your account. Please use the link below to reset your password.{1}
-inviteBodyHtml=<p> Hi {0}, </p> <p> You’ve been added to the UID2 Portal. </p> <p> Click the button to accept your invitation and create a password for your account. </p> <p>Note: This invitation expires in {2}.</p> <div class="footer"> <a href="{1}" class="button" >Accept Invitation</a > </div>
+inviteBodyHtml=<p> Hi {0}, </p> <p> You’ve been added to the UID2 Portal. </p> <p> Click the button to accept your invitation and create a password for your account. </p> <p> If you didn’t request access, you can ignore this message. </p> <p>Note: This invitation expires in {2}.</p> <div class="footer"> <a href="{1}" class="button" >Accept Invitation</a > <p> <br /> If the invitation has expired, <a href="http://localhost:3000/selfReinvite?email={3}" class="link">request a new invitation</a>. </p> </div>
```

Comment:
You still have `localhost:3000` in the template.

Reply:
Fixed!